### PR TITLE
LOG-1975: return back metrics service for CLO

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>

### Description

/cc 
/assign 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA: https://issues.redhat.com/browse/LOG-1975
- Enhancement proposal:

### CODE REVIEW

File: bundle/metadata/annotations.yaml [hunk 1]
```diff
@@ -1,7 +1,15 @@
 annotations:
-  operators.operatorframework.io.bundle.channel.default.v1: stable
-  operators.operatorframework.io.bundle.channels.v1: stable,stable-5.4
-  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  # Core bundle annotations.
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
-  operators.operatorframework.io.bundle.package.v1: cluster-logging
+  operators.operatorframework.io.bundle.package.v1: cluster-logging-operator
```
Comment (jcantrill) [inline bundle/metadata/annotations.yaml:6]: I think all these changes needed to be reverted given the package changed

File: config/prometheus/monitor.yaml [hunk 2]
```diff
@@ -5,12 +5,13 @@ kind: ServiceMonitor
...
 spec:
   endpoints:
     - path: /metrics
-      port: https
+      targetPort: 8686
+      port: 8686
   selector:
     matchLabels:
-      control-plane: controller-manager
+      control-plane: cluster-logging-operator
```
Comment (jcantrill) [inline config/prometheus/monitor.yaml:17]: does this match a label set on our operator?

File: manifests/5.4/cluster-logging-operator-metrics-monitor_monitoring.coreos.com_v1_servicemonitor.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  labels:
+    control-plane: controller-manager
+  name: cluster-logging-operator-metrics-monitor
+spec:
+  endpoints:
+  - path: /metrics
+    targetPort: 8686
+  selector:
+    matchLabels:
+      control-plane: cluster-logging-operator
```
Comment (jcantrill) [inline manifests/5.4/cluster-logging-operator-metrics-monitor_monitoring.coreos.com_v1_servicemonitor.yaml:13]: same here.. does this match labels on the operator

File: config/prometheus/monitor.yaml (hunk 2 above)
Comment (vparfonov) [reply config/prometheus/monitor.yaml:17]: yes, label the same as for operator pod

- service 
screenshot-console-openshift-console apps vparfonov-log1975x devcluster openshift com-2021 12 02-22_40_04

- pod 
screenshot-console-openshift-console apps vparfonov-log1975x devcluster openshift com-2021 12 02-22_40_56

File: bundle/metadata/annotations.yaml (hunk 1 above)
Comment (jcantrill) [reply bundle/metadata/annotations.yaml:6]: @vparfonov other then either: fixing the package name for this file or reverting it entirely, this PR looks good

Comment (vparfonov) [reply bundle/metadata/annotations.yaml]: done

**Pull request merged**